sql-plugin/pom.xml

```diff
@@ -460,6 +460,125 @@
+                                <dirset dir="${project.basedir}/src/main">
+                                    <include name="311+-nondb/*"/>
+                                    <include name="311until*/*"/>
+                                    <include name="pre320-treenode/*"/>
+                                </dirset>
+                            </pathconvert>
+                            <!-- TODO loop? -->
+                            <pathconvert property="spark311.sources" pathsep=",">
+                                <dirset dir="${project.basedir}/src/main">
+                                    <include name="311-nondb/*"/>
+                                </dirset>
+                            </pathconvert>
+                            <pathconvert property="spark312.sources" pathsep=",">
+                                <dirset dir="${project.basedir}/src/main">
+                                    <include name="312-nondb/*"/>
+                                </dirset>
+                            </pathconvert>
+                            <pathconvert property="spark313.sources" pathsep=",">
+                                <dirset dir="${project.basedir}/src/main">
+                                    <include name="313/*"/>
+                                </dirset>
+                            </pathconvert>
+                            <pathconvert property="spark314.sources" pathsep=",">
+                                <dirset dir="${project.basedir}/src/main">
+                                    <include name="314/*"/>
+                                </dirset>
+                            </pathconvert>
+                            <pathconvert property="spark32X.sources" pathsep=",">
+                                <dirset dir="${project.basedir}/src/main">
+                                    <include name="311+-nondb/*"/>
+                                    <include name="311until330*/*"/>
+                                    <include name="311until340*/*"/>
+                                    <include name="320until*/*"/>
+                                    <include name="320+*/*"/>
+                                    <include name="post320-treenode/*"/>
+                                </dirset>
+                            </pathconvert>
+                            <pathconvert property="spark320.sources" pathsep=",">
+                                <dirset dir="${project.basedir}/src/main">
+                                    <include name="320/*"/>
+                                </dirset>
+                            </pathconvert>
+                            <pathconvert property="spark321X.sources" pathsep=",">
+                                <dirset dir="${project.basedir}/src/main">
+                                    <include name="311+-nondb/*"/>
+                                    <include name="311until330*/*"/>
```

Comment:
isn't this spark320X.sources + some other?

Reply:
will redo this part. the idea was to build a dirset for each feature line 32X and then individual patch dirsets.